requester/requester.go

```diff
@@ -112,6 +118,10 @@ func (b *Work) Run() {
 	go func() {
 		runReporter(b.report)
 	}()
+	b.renderer = newRenderer(b.rendererCh)
```

Comment:
Might as well add a check on the switch before enabling the renderer at all.

Otherwise we're doing needless work.